apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/ClientPool.cs

```diff
@@ -44,17 +44,21 @@ internal static class ClientPool
             return result;
         }
 
-        // ReSharper disable once UnusedMember.Global
-        internal static void DumpPoolContents(StringBuilder s)
+        //ReSharper disable once UnusedMember.Global
+        //Returns true if all client reference counts are zero.
+        //For test purposes only.
+        internal static int GetTotalClientRefCount(StringBuilder poolContents)
```

Comment:
Nit: This method really does two things but the method name only implies one of them. Either add comments to explain why a `StringBuilder` needs to be specified and what will happen to it and/or change the signature to something like this:
```csharp
internal static void GetPoolDebugInfo(StringBuilder poolContents, out int refCount)
```